app/scripts/proactive/view/TaskView.js

```diff
@@ -37,7 +37,11 @@ define(
             var iconPath = this.icons[this.modelType];
             try{
 	            iconPath = this.iconsPerLanguage[this.model.get("Execute").get("Script").get("Language")];
-            }catch(err){}
+            }catch(err){
+            	 try{
+     	            iconPath = this.iconsPerLanguage[this.model.get("Execute").Script.Language];
```

Comment:
Why do we have those two different ways to access it. 

I guess, that when we initially create the model we create it differently compared to when we change the script. 
It seems that we solve the symptom but that is not really the issue.